content/docs/reference/pkg/nodejs/pulumi/kubernetes/admissionregistration/v1beta1/_index.md

```diff
@@ -44,7 +44,7 @@ or reject and may change the object.
 <div class="pdoc-member-contents">
 {{% md %}}
 
-<pre class="highlight"><span class='kd'></span><span class='kd'>new</span> MutatingWebhookConfiguration(name: <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String'>string</a></span>, args?: inputApi.admissionregistration.v1beta1.MutatingWebhookConfiguration, opts?: <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#CustomResourceOptions'>pulumi.CustomResourceOptions</a>)</pre>
+<pre class="highlight"><span class='kd'></span><span class='kd'>new</span> MutatingWebhookConfiguration(name: <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String'>string</a></span>, args?: <a href='/docs/reference/pkg/nodejs/pulumi/@pulumi/kubernetes/types/input/admissionregistration/v1beta1/#MutatingWebhookConfiguration'>inputs.admissionregistration.v1beta1.MutatingWebhookConfiguration</a>, opts?: <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#CustomResourceOptions'>pulumi.CustomResourceOptions</a>)</pre>
```

Comment:
The URL is including an extra `@pulumi` component that should be removed.

This:

```
/docs/reference/pkg/nodejs/pulumi/@pulumi/kubernetes/types/input/...
```

Should be:

```
/docs/reference/pkg/nodejs/pulumi/kubernetes/types/input/...
```

Reply:
Ahh - yes - this was actually fixed already but the kubernetes docs weren't regenerated with the fix.  Should be taken care of as part of regenerating all the docs.